app/code/Magento/Checkout/view/frontend/web/js/view/cart/shipping-estimation.js

```diff
@@ -88,6 +88,11 @@ define(
                     }
                 });
 
+                //To enable proceed to checkout button after initializing shipping form
+                if (window.checkoutConfig.isAllowedToCheckout) {
```

Comment:
Shouldn't we check if this checking out is restricted?

Reply:
@Stepa4man  Yes, we need to check whether the checking out is restricted. 
Because when the subtotal of the cart is less than the Minimum amount to checkout, then the button should be disabled.

Reply:
@Stepa4man There is no issue on going to checkout without shipping estimation section loaded.
But this fix will improves the user experience, If the button is disabled until it is loaded, then the user will get to know some section needs to be; render.
Right now user is clicking the Proceed to checkout button even before the section loads. Because shipping section is taking more time to load.

Reply:
> @Stepa4man Yes, we need to check whether the checking out is restricted.
> Because when the subtotal of the cart is less than the Minimum amount to checkout, then the button should be disabled.

I'm asking how this code checks if the checkout is restricted. 
For example with 'checkout/options/onepage_checkout_enabled' configuration

Reply:
@Stepa4man Currently this code is not checking for whether the one-page checkout is enabled or not. Do you want me to check that condition also?
What is your suggestion on this PR? 
Please give your feedback. I will improve the code.

Reply:
I personally don't understand why should we change here something. That's why I'm asking questions.